src/Microsoft.AspNetCore.SignalR.Common/Protocol/HubProtocolExtensions.cs

```diff
@@ -30,9 +30,15 @@ public static byte[] GetMessageBytes(this IHubProtocol hubProtocol, HubMessage m
             }
         }
 
+        /// <summary>
+        /// Gets the minor version for an <see cref="IHubProtocol"/>.
+        /// Defaults to 0 if the protocol does not specify a minor version.
+        /// </summary>
+        /// <param name="protocol"></param>
+        /// <returns></returns>
```

Comment:
The protocol's minor version. Defaults to 0 if the protocol does not specify a minor version.